Expensify/App
# [HOLD translations] Add existing owners error modal during bank account setup

### Details
Previously, `setupWithdrawalAccount` would silently error when a user tried to add a withdrawal bank account that we already assigned an owner to. 

This PR adds some text similar to [the text in Web-Secure](https://github.com/Expensify/Web-Secure/blob/2f20e1b0b13b2d09081ecf0986a64a8af4a0a67f/site/app/dialogs/reimbursementAccount/existingCopiesWarning.jsx#L3-L49).

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/169896

### Tests/QA
1. Create a new workspace and click "Get Started".
2. Add a bank account manually following the steps [here](https://stackoverflow.com/c/expensify/questions/342).
3. In a different account, follow steps 1 and 2.
4. Verify that you're unable to add the account since the bank account already has an existing owner, and verify that the error step displays (see screenshots).

### Tested On


### Screenshots
_NOTE: I've since updated the header title to be capitalized [aka `Existing Owners`] (not shown in the screenshots below)_


#### Desktop
<img width="1203" alt="Screen Shot 2021-07-13 at 3 32 18 PM" src="https://user-images.githubusercontent.com/31285285/125412178-3e7dfe00-e3f1-11eb-86fb-87a780f70fd5.png">

#### Web
<img width="1792" alt="Screen Shot 2021-07-13 at 3 34 47 PM" src="https://user-images.githubusercontent.com/31285285/125412183-4047c180-e3f1-11eb-9445-f2a9963468ce.png">

#### Mobile-Web
<img width="374" alt="Screen Shot 2021-07-13 at 3 36 19 PM" src="https://user-images.githubusercontent.com/31285285/125412191-84b60644-75d5-4e37-862d-361420c7885f.png">

#### Android
![WechatIMG26](https://user-images.githubusercontent.com/31285285/125412562-9ddc0e00-e3f1-11eb-92b2-d1efa6567b31.jpeg)

cc @marcaaron Can you verify if the testing steps are correct? I was unable to test entirely on dev because I can't hit the Web-Secure API.
